libcxx/include/__algorithm/stable_sort.h

```diff
@@ -133,20 +138,24 @@ _LIBCPP_HIDE_FROM_ABI void __merge_move_assign(
     *__result = _Ops::__iter_move(__first2);
 }
 
-template <class _AlgPolicy, class _Compare, class _RandomAccessIterator>
-void __stable_sort(_RandomAccessIterator __first,
-                   _RandomAccessIterator __last,
-                   _Compare __comp,
-                   typename iterator_traits<_RandomAccessIterator>::difference_type __len,
-                   typename iterator_traits<_RandomAccessIterator>::value_type* __buff,
-                   ptrdiff_t __buff_size);
+template <class _AlgPolicy, class _Compare, class _RandomAccessIterator, bool _EnableRadixSort>
+void __stable_sort(
+    _RandomAccessIterator __first,
+    _RandomAccessIterator __last,
+    _Compare __comp,
+    typename iterator_traits<_RandomAccessIterator>::difference_type __len,
```

Comment:
```suggestion
    __iter_diff_t<_RandomAccessIterator> __len,
```

Reply:
This is the code I didn't changed, just reformatted. Should I change to `__iter_diff_t` and `__iter_value_type` anyway?

Reply:
Hmm. This code should already be formatted correctly. Do you maybe have a clang-format that's too old?

Reply:
I'm using

```
$ clang-format --version
clang-format version 18.1.8
```

and the following command called from the repo root.
```
git diff -U0 --no-color my/main | clang/tools/clang-format/clang-format-diff.py -style=file:libcxx/.clang-format -p1 -i
```